composer.json

```
@@ -41,39 +42,41 @@
         "elasticsearch/elasticsearch": "^7.17||^8.5",
         "ezyang/htmlpurifier": "^4.14",
         "guzzlehttp/guzzle": "^7.4",
-        "laminas/laminas-captcha": "^2.12",
-        "laminas/laminas-code": "^4.5",
-        "laminas/laminas-db": "^2.15",
-        "laminas/laminas-di": "^3.7",
+        "laminas/laminas-captcha": "^2.16",
+        "laminas/laminas-code": "^4.8",
+        "laminas/laminas-db": "^2.17",
+        "laminas/laminas-di": "^3.12",
         "laminas/laminas-escaper": "^2.10",
         "laminas/laminas-eventmanager": "^3.5",
         "laminas/laminas-feed": "^2.17",
-        "laminas/laminas-file": "^2.11",
+        "laminas/laminas-file": "^2.12",
         "laminas/laminas-filter": "^2.17",
         "laminas/laminas-http": "^2.15",
         "laminas/laminas-i18n": "^2.17",
         "laminas/laminas-mail": "^2.16",
         "laminas/laminas-mime": "^2.9",
         "laminas/laminas-modulemanager": "^2.11",
         "laminas/laminas-mvc": "^3.3",
-        "laminas/laminas-oauth": "^2.4",
+        "laminas/laminas-oauth": "^2.5",
         "laminas/laminas-permissions-acl": "^2.10",
+        "laminas/laminas-recaptcha": "^3.6",
         "laminas/laminas-servicemanager": "^3.16",
-        "laminas/laminas-soap": "^2.10",
+        "laminas/laminas-session": "^2.16",
+        "laminas/laminas-soap": "^2.11",
         "laminas/laminas-stdlib": "^3.11",
         "laminas/laminas-uri": "^2.9",
         "laminas/laminas-validator": "^2.23",
         "league/flysystem": "^2.4",
         "league/flysystem-aws-s3-v3": "^2.4",
-        "magento/composer": "^1.9.0-beta1",
+        "magento/composer": "^1.9",
         "magento/composer-dependency-version-audit-plugin": "^0.1",
         "magento/magento-composer-installer": ">=0.4.0-beta1",
         "magento/zend-cache": "^1.16",
         "magento/zend-db": "^1.16",
         "magento/zend-pdf": "^1.16",
         "monolog/monolog": "^2.7",
         "opensearch-project/opensearch-php": "^1.0 || ^2.0",
-        "pelago/emogrifier": "^6.0.0",
+        "pelago/emogrifier": "^7.0",
```

What's not compatible with version `6.0` in this pull request?
```suggestion
        "pelago/emogrifier": "^6.0 || ^7.0",
```

---

Feels like it's already upgraded like that in 2.4-develop, so it make sense to keep as it was.